minishift-ci-index.yaml

```diff
@@ -246,6 +246,14 @@
                 - master
     triggers:
         - timed: '0 0 * * *'
+    publishers:
+        - email-ext:
+            recipients: minishift@lists.minishift.io
+            content-type: html
+            subject: Minishift nightly build ${BUILD_NUMBER}
+            body: "The build has finished. Build URL: ${BUILD_URL}"
+            failure: true
+            success: true
```

Comment:
@budhrg you want mail for both failure and success here. I think we should only mail if job failed?

Reply:
@budhrg Can we put the email address of maintainers first instead of `minishift@lists.minishift.io`? Once we get confidence about the content of the email we can put minishift@lists.minishift.io. Personally I am not comfortable with sending something to minishift@lists.minishift.io if I am not sure what would the content look like.

Reply:
on failure should be good enough

Reply:
The end game should be to the mailing list. I think it should be save to try, since the email would anyways end up in a special queue, since the sender would not be registered. 

That said, nothing wrong with trying with a different email first to see how the email looks like and from whom it gets send.

Reply:
@LalatenduMohanty , this will be content I believe

```
The build has finished. Build URL: https://ci.centos.org/job/minishift-nightly/10/
```
and

Subject: 

```
Minishift nightly build 10
```